src/Compilers/Core/Portable/PEWriter/MetadataWriter.cs

```diff
@@ -899,27 +899,6 @@ private EntityHandle GetExportedTypeImplementation(INamespaceTypeReference names
                 : GetAssemblyReferenceHandle(aref);
         }
 
-        private static uint GetManagedResourceOffset(ManagedResource resource, BlobBuilder resourceWriter)
-        {
-            if (resource.ExternalFile != null)
-            {
-                return resource.Offset;
-            }
-
-            int result = resourceWriter.Count;
-            resource.WriteData(resourceWriter);
-            return (uint)result;
-        }
-
-        private static uint GetManagedResourceOffset(BlobBuilder resource, BlobBuilder resourceWriter)
-        {
-            int result = resourceWriter.Count;
-            resourceWriter.WriteInt32(resource.Count);
-            resource.WriteContentTo(resourceWriter);
-            resourceWriter.Align(8);
-            return (uint)result;
-        }
```

Comment:
These methods are only used in `PopulateManifestResourceTableRows`. Given the tight relationship that has to the impl details of these methods now I thought it better to make them local functions instead.